src/app/components/dynamicdialog/dynamicdialog.ts

```diff
@@ -266,6 +267,13 @@ export class DynamicDialogComponent implements AfterViewInit, OnDestroy {
         return this.config?.templates?.closeicon;
     }
 
+    get dynamicDialogCount() {
+        let dynamicDialogs = this.document.querySelectorAll('p-dynamicdialog');
```

Comment:
You can use `const` instead of `let` because the variables will not be reasigned